app/serializers/questions_serializer.rb

```diff
@@ -4,9 +4,13 @@ class QuestionsSerializer < ActiveModel::Serializer
   attribute :version_independent_id, key: :questionId
   attribute :content, key: :questionText
   attribute :question_uri, key: :questionUri
-  attribute :version, key: :version
-  attribute :published_by, serializer: UserSerializer
   def question_uri
     Rails.application.routes.url_helpers.api_question_url(object.version_independent_id, version: object.version, only_path: true)
   end
+  attribute :version, key: :version
+  attribute :published_by, serializer: UserSerializer
+  attribute :question_type, key: :questionType
+  def question_type
+    object.question_type.name
+  end
```

Comment:
Not a big deal, but the methods mixed with the attributes seems odd. If the desire is to have the implementation right next to the attribute, I'd recommend passing a block to attribute, there is an example in the docs:
http://www.rubydoc.info/github/rails-api/active_model_serializers/ActiveModel/Serializer#attribute-class_method